src/main/java/net/dv8tion/jda/client/entities/impl/GroupImpl.java

```diff
@@ -44,6 +45,9 @@
     private String name;
     private String iconId;
     private long lastMessageId;
+	private GroupManager manager;
+	
+	private final Object mngLock = new Object();
```

Comment:
Some people only use the updatable manager which would mean we would construct the facade for no reason.